pkg/katautils/config.go

```diff
@@ -700,6 +700,96 @@ func newAcrnHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
 	}, nil
 }
 
+func newIchHypervisorConfig(h hypervisor) (vc.HypervisorConfig, error) {
+	hypervisor, err := h.path()
```

Comment:
As I look at the code in general in this file, a lot of the code to initialise hypervisor config is being duplicated across hypervisors. If not this PR, we should consider pulling all the common code together.

Reply:
I see your point but prefer to do this after I've finished this one.